### PR TITLE
Configured to use apollo graphql endpoint

### PR DESCRIPTION
**Pull request by colw**
Closes #ISSUE

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/components/common/TmConnectedNetwork.vue [hunk 1]
```diff
@@ -87,6 +87,10 @@ export default {
         return `Seeking connection`
       }
     }
+  },
+  mounted() {
+    console.log(this.connection.network)
```
Comment (faboweb) [inline src/components/common/TmConnectedNetwork.vue]: Should we keep this in?

File: src/components/governance/PageProposal.vue [hunk 2]
```diff
@@ -150,85 +154,99 @@ export default {
...
     },
     abstainPercentage({ tally, totalVotes } = this) {
-      return num.percentInt(totalVotes === 0 ? 0 : tally.abstain / totalVotes)
+      return num.percentInt(totalVotes === 0 ? 0 : tally.final_tally_abstain / totalVotes)
     },
-    tally({ proposals, proposalId } = this) {
-      const { yes, no, abstain, no_with_veto } =
-        proposals.tallies[proposalId] || {}
+    tally() {
+      const {
```
Comment (faboweb) [inline src/components/governance/PageProposal.vue]: I guess we need to rework this, once the data schema is adjusted to standardize current tally and final tally.

File: src/router.js [hunk 3]
```diff
@@ -44,11 +44,9 @@ export default router
...
+  const feature = `feature_${to.meta.feature.toLowerCase()}`
   const { data } = await apollo.query({
-    query: NetworkCapability(
-      networkId,
-      `feature_${to.meta.feature.toLowerCase()}`
-    )
+    query: NetworkCapability(networkId)
   })
-  return NetworkCapabilityResult(data)
+  return NetworkCapabilityResult(feature)(data)
```
Comment (faboweb) [inline src/router.js]: 👍 I guess this is better for caching

File: src/App.vue [hunk 4]
```diff
@@ -2,7 +2,7 @@
   <div>
     <div id="bar-container">
       <CookieBar />
-      <MaintenanceBar />
```
Comment (jbibla) [inline src/App.vue]: why did we remove this?

Comment (faboweb) [reply src/App.vue]: this was just merged into a feature branch and needs several fixes before we merge it to develop

**Pull request merged**